<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>530</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>18</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Добавление записи</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Порода</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_7"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Группа диаметров</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_12"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_15"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Тех. годность</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_13">
          <item alignment="Qt::AlignLeft">
           <widget class="QRadioButton" name="radioButton_34">
            <property name="text">
             <string>Деловая</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QRadioButton" name="radioButton_35">
            <property name="text">
             <string>Дровяная</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignLeft">
           <widget class="QRadioButton" name="radioButton_36">
            <property name="text">
             <string>Биоразнообразие</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Колличество: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="horizontalSlider">
         <property name="minimumSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="pageStep">
          <number>1</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>1</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_2</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>323</y>
    </hint>
    <hint type="destinationlabel">
     <x>641</x>
     <y>323</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
